myhpi/static/scss/myHPI.scss

```diff
@@ -5,20 +5,25 @@
 $primary: #F5631E;
 $secondary: #F5631E;

```

Comment:
In `[myhpi/core/templates/core/sidebar.html](https://github.com/fsr-de/myHPI/pull/525/files/bc3f7996429831f3857afe63e7b344cfd023d219#diff-8c66ac3d840d267494b184c335c50d73e56b70724f454183f3af2b86242632a7)`, there is the following comment:
```
<!-- Bootstrap 5.2 supports responsive offcanvas. This duplication is a workaround and should be removed after the upgrade -->
```

Could be addressed now, since we updated to 5.3